### PR TITLE
(fix)O3-2579 Lab results: Too much grey space

### PR DESCRIPTION
**Pull request by suubi-joshua**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [ ] My work includes tests or is validated by existing tests.

## Summary
Removed the background color of the results grid.

## Related Issue

https://openmrs.atlassian.net/browse/O3-2579

## Other
[Screencast from 07-17-2024 12:37:13 AM.webm](https://github.com/user-attachments/assets/b5fc3490-0cb2-4889-9b0e-7bb80cc3d55b)

[Screencast from 07-17-2024 09:51:57 PM.webm](https://github.com/user-attachments/assets/efa76dc5-62d7-43e8-97c4-85e8cdcdb0b0)

### CODE REVIEW

**Comment by suubi-joshua** [comment]
@sherrif10 here is a cleaner PR

File: packages/esm-patient-labs-app/src/test-results/panel-timeline/timeline.scss [hunk 1]
```diff
@@ -75,6 +74,9 @@
   background-color: $color-gray-30;
   grid-template: auto auto / 9em auto;
   border-collapse: collapse;
+  
+  width: fit-content;
```
Comment (sherrif10) [inline packages/esm-patient-labs-app/src/test-results/panel-timeline/timeline.scss:78]: Looks good to me. Thanks @suubi-joshua   , cc @denniskigen

**Review by sherrif10 — APPROVED** [review]

File: packages/esm-patient-labs-app/src/test-results/panel-timeline/timeline.scss (hunk 1 above)
Comment (NethmiRodrigo) [reply packages/esm-patient-labs-app/src/test-results/panel-timeline/timeline.scss:78]: Shouldn't all panels consistently take up the entire available space without fitting to the individual content sizes?
More so like:
Screenshot 2024-07-19 at 7 03 00 PM

Comment (suubi-joshua) [reply packages/esm-patient-labs-app/src/test-results/panel-timeline/timeline.scss:78]: Yes this looks more neat. @NethmiRodrigo

File: packages/esm-patient-labs-app/src/test-results/panel-timeline/timeline.scss [hunk 2]
```diff
@@ -75,6 +74,9 @@
   background-color: $color-gray-30;
   grid-template: auto auto / 9em auto;
   border-collapse: collapse;
+
```
Comment (brandones) [inline packages/esm-patient-labs-app/src/test-results/panel-timeline/timeline.scss:77]:
```suggestion
```

**Comment by brandones** [comment]
I assume the two videos in the description should be labeled "Before" and "After."

**Review by brandones — APPROVED** [review]

**Pull request merged**